_monitoring-your-cluster/pa/rca/shard-hotspot.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Shard hotspot identification
+parent: Root Cause Analysis
+grand_parent: Performance Analyzer
+nav_order: 30
+---
+
+## Shard hotspot identification
+
+With the shard hotspot identification RCA, you can identify a hot shard within an index. A hot shard is an outlier that consumes more resources than its counterparts. The shard hotspot identification RCA monitors the following metrics:
+
+- `CPU_Utilization`
+- `Heap_AllocRate`
+
+These metrics provide an accurate picture of operation intensities for certain shards, such as the following: 
```

Comment:
I think we should mention here that although the metric that you should monitor depends on the cluster configuration and the workload, these are suggested metrics to monitor for the following operations.

Reply:
So, my understanding here is that these are the two metrics specifically monitored by this RCA, and only these two. Are you saying this isn't correct?

Reply:
No, I understand that those are the two metrics, but the intro sentence here can be rewritten to be more explicit. The question is,  how does the user know which metric to monitor? That depends on the user's cluster configuration and workload. However, for let's say for workloads with heavy bulk operation usage, we recommend that they look for the high heap allocation rate. For workloads with heavy search request usage, they should look for high CPU utilization, etc.

Reply:
Or, another way it can be written is to emphasize which operations may lead to which resource consumption.

Reply:
ah, ok, I'll work on it